src/reducers/paymentsReducer.js

```diff
@@ -9,7 +9,7 @@ const paymentReducer = createReducer(initialState, {
   [GET_PAYMENTS](state, action) {
     return {
       ...state,
-      payments: action.data.payments
+      payments: [...action.data.payments]
```

Comment:
what is the consequence of this change?

Reply:
It's making a copy of the array. Redux works with immutable objects in order to detect changes. There were some issues with that, so we changed this.